homeassistant/components/miele/climate.py

```diff
@@ -192,9 +197,11 @@ def __init__(
     @property
     def target_temperature(self) -> float | None:
         """Return the target temperature."""
-        if self.entity_description.target_fn(self.device) is None:
-            return None
-        return cast(float | None, self.entity_description.target_fn(self.device))
+        return (
+            cast(float | None, self.entity_description.target_fn(self.device))
+            if self.entity_description.target_fn(self.device) is not None
+            else None
+        )
```

Comment:
Let's assign it to a variable first, maybe use a walrus operator to make sure we don't execute the function twice

Reply:
Something like this?